examples/grafana-metrics/Makefile

```diff
@@ -1,3 +1,6 @@
+OS_RELEASE:=ubuntu
```

Comment:
These 3 variables aren't being used so that can be removed

Reply:
I updated the PR to use the variables